examples/repo_example/basic_repo.py

```diff
@@ -332,7 +333,7 @@ def _in(days):
 del roles["targets"].signed.targets[target_path]
 
 # Increase expiry (delegators should be less volatile)
-roles["targets"].expires = _in(365)
+roles["targets"].signed.expires = _in(365)
```

Comment:
👀 Whoopsi, thanks for catching this! Did mypy detect it?

Reply:
Yes, it did.